### PR TITLE
Updated makefile build command steps

### PR DESCRIPTION
**Pull request by prenx4x**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ init:
 	pip install -Ur requirements.txt
 
 build:
-	build-image-src/build_all_images.sh
+	cd build-image-src; ./build_all_images.sh
```
Comment (hoffa) [inline Makefile]: nit: similar style to other commands?
```suggestion
	cd build-image-src
	./build_all_images.sh
```

**Review by hoffa — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (prenx4x) [reply Makefile]: this doesn't work in makefile - https://stackoverflow.com/questions/1789594/how-do-i-write-the-cd-command-in-a-makefile

Comment (hoffa) [inline Makefile]: So that the build isn't executed if `cd` fails:

```suggestion
	cd build-image-src && ./build_all_images.sh
```

Comment (hoffa) [reply Makefile]: Interesting behavior, wasn't aware!

**Review by awood45 — APPROVED** [review]

**Pull request merged**